proposals/lip-0058.md

```diff
@@ -763,8 +763,6 @@ In the before application processing stage of a genesis block `b`, the BFT store
   * `bftVotes.maxHeightCertified = b.header.height`,
   * `bftVotes.blockBFTInfos` is an empty array,
   * `bftVotes.activeValidatorsVoteInfo` is an empty array.
-* The variable `genesisHeight` is initialized as follows:
-  * `genesisHeight = b.header.height`.
 
 #### After Application Processing
 
```

Comment:
Question about lines 779-780: Those two steps actually suggest a queue data structure where new entries are added to the one side and old entries are deleted from the other side (it would be inefficient to use a list/array since at each step need to re-assign all indexes). I was just curious about how this is actually implemented..

Reply:
As it is outside the scope of this LIP update, it could be clarified on Slack? Note for the serialized data we cannot use a queue.

Reply:
Sounds good, it's anyways more of a curiosity question.